### ISSUE TITLE
rewrite kerberos-authentication integration

### ISSUE THREAD

**Issue by Pajinek** [SatelliteQE/GreenTea#7]
more information is on the page http://www.adelton.com/docs/apache/optional-kerberos-authentication

**Comment by Pajinek** [comment]
Currently it should be fixed from version 1.1.0 (using Django 1.10)

```
>> kinit
>> curl -I --negotiate -u :     http://<FQDN>/accounts/login
```

**Comment by Pajinek** [comment]
Configuration for Apacha2 with mod_proxy:

```
...
<Location "/accounts/login">
             AuthType Kerberos
             AuthName "Grean Tea - Kerberos login"
             KrbMethodNegotiate on
             #KrbAuthoritative on
             KrbServiceName Any
             KrbAuthRealm DOMAIN.COM
             Krb5Keytab /etc/httpd/conf/httpd.tttt.keytab
             Require valid-user
 </Location>
...
    ProxyPass / http://localhost:8000/
    ProxyPassReverse / http://localhost:8000/
    RequestHeader set REMOTE-USER %{REMOTE_USER}s
```
